sonic_sfp/sfputilbase.py

```diff
@@ -5,22 +5,22 @@
 #
 #--------------------------------------------------------------------------
 try:
-	import fcntl
-	import struct
-	import sys
-	import time
-	import binascii
-	import os
-	import getopt
-	import re
-	import bcmshell
-	import pprint
-	import eeprom_dts
-	from math import log10
-	from sff8472 import sff8472InterfaceId
-	from sff8472 import sff8472Dom
-	from sff8436 import sff8436InterfaceId
-	from sff8436 import sff8436Dom
+    import fcntl
+    import struct
+    import sys
+    import time
+    import binascii
+    import os
+    import getopt
+    import re
+    import bcmshell
+    import pprint
+    from math import log10
+    from sonic_eeprom import eeprom_dts
+    from sff8472 import sff8472InterfaceId
+    from sff8472 import sff8472Dom
+    from sff8436 import sff8436InterfaceId
+    from sff8436 import sff8436Dom
```

Comment:
Already done. Since sfputilbase, sff8436 and sff8472 all reside in the same package, they can reference one another using relative paths. I could make the paths absolute for clarity, if desired.